docs/cli_installation.md

```diff
@@ -49,24 +55,7 @@ brew install argocd
 
 ### Download With Curl
 
-You can view the latest version of Argo CD at the link above or run the following command to grab the version:
-
-```bash
-VERSION=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
completely removed this, there is now a `latest` tag, so I didn't really see the value in retaining it
(and if someone wants to install a specific concrete version they can just go browse the releases)
--
this is now reflected in the Linux curl steps